### PR TITLE
(FACT-1615) move regression tests into custom_facts

### PR DESCRIPTION
**Pull request by ericldelaney**
Moved 2 regression tests into custom_facts
Update test that verifies how mco loads facter.rb
Updated for code style, Test Rail ID, and risk tags

### CODE REVIEW

**Comment by ericldelaney** [comment]
Tested on all platforms.

**Comment by ericldelaney** [comment]
@johnduarte I looked into how mco loads facter and I've updated the test acceptance/tests/load_libfacter.rb with more comments about what its doing as it matches how mco loads facter.rb

File: acceptance/tests/load_libfacter.rb [hunk 1]
```diff
@@ -1,31 +1,66 @@
...
+    teardown do
+      on(agent, "rm -rf '#{fact_dir}'")
+    end
+
+    if agent['platform'] =~ /windows/
+      # on Windows we have to figure out where facter.rb is so we can include the path
+      # figure out the root of the Puppet installation
+      puppet_ruby_path = on(agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby").stdout.chomp
+      cygwin_puppet_root = puppet_ruby_path_to_puppet_install_dir(puppet_ruby_path)
+      puppet_root = on(agent, "cygpath -w '#{cygwin_puppet_root}'").stdout.chomp
```
Comment (johnduarte) [inline acceptance/tests/load_libfacter.rb]: @ericldelaney it is unfortunate that we need all of this jiggery-pokery just to get the root install directory of `puppet-agent`. Can you file a Beaker ticket to have such a facility added to Beaker core?

**Comment by ericldelaney** [comment]
@LuvCurves can you review this?

**Comment by Magisus** [comment]
I like moving things into subdirectories, it's easier to get a feel for the layout of the tests 👍

**Pull request merged**